### ISSUE TITLE
HTTP Error 403: Forbidden with first email

### ISSUE THREAD

**Issue by mmoomocow** [sendgrid/sendgrid-python#885]
### Issue Summary
I was setting up sendgrid with python and followed the [sendgrid guide](https://app.sendgrid.com/guide/integrate/langs/python/verify), then when I tried to send the first email I got the error
### Steps to Reproduce
1. Install sendgrid package (pip install sendgrid)
2. paste in code and change to correct variables

### Code Snippet
```python
# using SendGrid's Python Library
# https://github.com/sendgrid/sendgrid-python
from sendgrid import SendGridAPIClient
from sendgrid.helpers.mail import Mail

message = Mail(
    from_email='EMAIL',
    to_emails='EMAIL',
    subject='Sending with Twilio SendGrid is Fun',
    html_content='<strong>and easy to do anywhere, even with Python</strong>')
try:
    sg = SendGridAPIClient(
        'KEY'
        )
    response = sg.send(message)
    print(response.status_code)
    print(response.body)
    print(response.headers)
except Exception as e:
    print(e)
```

### Exception/Log
```
HTTP Error 403: Forbidden
```

### Technical details:
* sendgrid-python version: 3.2.7
* python version: 3.6.8

**Comment by jaimemarijke** [comment]
I am also running into this

**Comment by childish-sambino** [comment]
@mmoomocow Recommend upgrading to a later release of this library. I'm not even seeing a `3.2.7` version.

@jaimemarijke Are you seeing this when performing the same mail send operation?

**Comment by jaimemarijke** [comment]
Yes, I got the 403 after following these instructions: https://app.sendgrid.com/guide/integrate/langs/python

I use pyenv to set the python version in my repo, and tox to create a python virtual environment (`py35`)

```bash
╭─(py35) jaime@kulshan /Users/jaime/personal/small-questions-email  <master*>
╰─$ python --version
Python 3.5.9
╭─(py35) jaime@kulshan /Users/jaime/personal/small-questions-email  <master*>
╰─$ pip freeze | ag sendgrid
sendgrid==6.2.2
╭─(py35) jaime@kulshan /Users/jaime/personal/small-questions-email  <master*>
╰─$ printenv | ag SENDGRID                                                                                                                                                                                                               
SENDGRID_API_KEY=<my key>
╭─(py35) jaime@kulshan /Users/jaime/personal/small-questions-email  <master*>
╰─$ cat small_questions_email/sendgrid.py

# using SendGrid's Python Library
# https://github.com/sendgrid/sendgrid-python
import os
from sendgrid import SendGridAPIClient
from sendgrid.helpers.mail import Mail

message = Mail(
    from_email='from_email@example.com',
    to_emails='to@example.com',
    subject='Sending with Twilio SendGrid is Fun',
    html_content='<strong>and easy to do anywhere, even with Python</strong>')
try:
    sg = SendGridAPIClient(os.environ.get('SENDGRID_API_KEY'))
    response = sg.send(message)
    print(response.status_code)
    print(response.body)
    print(response.headers)
except Exception as e:
    print(e.message)
╭─(py35) jaime@kulshan /Users/jaime/personal/small-questions-email  <master*>
╰─$ python -m small_questions_email.sendgrid
Traceback (most recent call last):
  File "/Users/jaime/personal/small-questions-email/small_questions_email/sendgrid.py", line 15, in <module>
    response = sg.send(message)
  File "/Users/jaime/personal/small-questions-email/.tox/py35/lib/python3.5/site-packages/sendgrid/sendgrid.py", line 95, in send
    response = self.client.mail.send.post(request_body=message.get())
  File "/Users/jaime/personal/small-questions-email/.tox/py35/lib/python3.5/site-packages/python_http_client/client.py", line 262, in http_request
    self._make_request(opener, request, timeout=timeout)
  File "/Users/jaime/personal/small-questions-email/.tox/py35/lib/python3.5/site-packages/python_http_client/client.py", line 178, in _make_request
    raise exc
python_http_client.exceptions.ForbiddenError: HTTP Error 403: Forbidden

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/jaime/.pyenv/versions/3.5.9/lib/python3.5/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/Users/jaime/.pyenv/versions/3.5.9/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/Users/jaime/personal/small-questions-email/small_questions_email/sendgrid.py", line 20, in <module>
    print(e.message)
AttributeError: 'ForbiddenError' object has no attribute 'message'
```

**Comment by devsetgo** [comment]
I had the same issue until I finally got a user email registered. So you might need to check to see if the domain or sender email is validate on the dashboard.

I also used the without helper class example.

Good luck.

**Comment by childish-sambino** [comment]
+1 to @devsetgo's comment. @jaimemarijke Let me know if that doesn't resolve it for you.

**Comment by childish-sambino** [comment]
Also, try changing the exception logic to log the response body for more details:
```
print(e)
print(e.body)
```

**Comment by thiagolara** [comment]
Hi there folks, how do I check my account on dashboard? @childish-sambino  @devsetgo

**Comment by childish-sambino** [comment]
@thiagolara Have you tried logging the response body for additional error details?

**Comment by thiagolara** [comment]
Hey @childish-sambino , yes I just did it and I got this message:

b'{"errors":[{"message":"The from address does not match a verified Sender Identity. Mail cannot be sent until this error is resolved. Visit https://sendgrid.com/docs/for-developers/sending-email/sender-identity/ to see the Sender Identity requirements","field":"from","help":null}]}'

I was using my customer's email as sender in a contact form. 

I solved the problem changing the "from email" (sender email) to my own account email, instead of using my customer's email :)
